### PR TITLE
LPS-175279 Fix test integration

### PR DESCRIPTION
**Pull request by murilostodolni**
Related: https://issues.liferay.com/browse/LPS-175279

### CODE REVIEW

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java [hunk 1]
```diff
@@ -320,6 +319,8 @@ public void testUpdateDefaultObjectStateFlow() throws Exception {
 
 	@Test
 	public void testUpdateObjectStateTransitions() throws Exception {
+		_step4ListTypeEntry = _addListTypeEntry("step4");
+
```
Comment (carolmariaabb) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java:323]: @murilostodolni can we make it local? As we are only using it in this test.

```
ListTypeEntry listTypeEntry = _addListTypeEntry("step4");
```

Comment (carolmariaabb) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java:323]: @murilostodolni can we move it to be right before the `for (ObjectState objectState : objectStates) ` [here](https://github.com/liferay-objects/liferay-portal/blob/b2bf5d3409d13641d0fa87654b71302599037054/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java#L342)? Or the order matters here?

See the [As Used](https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752519/Java+conventions+in+Liferay#%5BinlineExtension%5DLocal-variable-declarations) topic for more context

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java (hunk 1 above)
Comment (murilostodolni) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java:323]: Perfect. I saved the link to read it all. Thank you @carolmariaabb

Comment (murilostodolni) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java:323]: Yes! Very well observed. Thanks

Comment (murilostodolni) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java:323]: @carolmariaabb We need to add the listTypeEntry before the get [here](We need to add the listTypeEntry before the get [here](https://github.com/liferay-objects/liferay-portal/blob/b2bf5d3409d13641d0fa87654b71302599037054/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java#L342:~:text=%22step4%22)). If we add after, [this](https://github.com/liferay-objects/liferay-portal/blob/b2bf5d3409d13641d0fa87654b71302599037054/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java#L342:~:text=_assertEquals(,.getObjectStateFlowId()))%3B) assert failed.

Comment (carolmariaabb) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java:323]: No problem, @murilostodolni .. let's make sure Brian doesn't break this kkkkk

**Comment by murilostodolni** [comment]
ci:test:sf

**Comment by murilostodolni** [comment]
ci:test:relevant

**Comment by murilostodolni** [comment]
Forwarding manually: https://github.com/brianchandotcom/liferay-portal/pull/130073

**Comment by murilostodolni** [comment]
ci:close

**Pull request closed without merging**